package-lock.json

```diff
@@ -121,31 +127,27 @@
     "acorn": {
       "version": "5.7.3",
       "resolved": "https://registry.npmjs.org/acorn/-/acorn-5.7.3.tgz",
-      "integrity": "sha512-T/zvzYRfbVojPWahDsE5evJdHb3oJoQfFbsrKM7w5Zcs++Tr257tia3BmMP8XYVjp1S9RZXQMh7gao96BlqZOw==",
-      "dev": true
+      "integrity": "sha512-T/zvzYRfbVojPWahDsE5evJdHb3oJoQfFbsrKM7w5Zcs++Tr257tia3BmMP8XYVjp1S9RZXQMh7gao96BlqZOw=="
     },
     "acorn-jsx": {
       "version": "3.0.1",
-      "resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
+      "resolved": "http://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
```

Comment:
As a minor comment, I find it strange that it goes from https to http here. Looks like something that should be fixed in the npm tooling.

Reply:
Yeah, that's a [known npm bug](https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285). It doesn't really matter because npm always uses HTTPS for fetching from the registry, but they're still trying to track down why it happens.